meson.build

```diff
@@ -36,7 +36,11 @@ add_project_arguments(
 zlib = dependency('zlib')
 threads = dependency('threads')
 jansson = dependency('jansson', version: '>=2.10')
-libcrypto = dependency('libcrypto', version: '>=1.0.2')
+if host_machine.system() == 'freebsd'
+  libcrypto = meson.get_compiler('c').find_library('crypto', dirs: [ '/usr/local/lib/','/usr/lib/' ])
```

Comment:
Is there a way that CPPFLAGS/LDFLAGS could be specified instead to find the library?

Reply:
I wish I knew.  I don't have a lot of experience with meson.  I would have thought the find_library call would find libcrypto without any extra dirs, but that doesn't seem to work on FreeBSD where sometimes the library is in /usr/lib (if default) and sometimes in /usr/local/lib (if an SSL out of the ports tree is used.)  I am MORE than happy to clean it up if you have a suggestion that works because I think it's a little ugly.  I couldn't figure out anything better.